src/svg-to-vectordrawable.js

```diff
@@ -639,6 +648,9 @@ JS2XML.prototype.refactorData = function(data, floatPrecision, fillBlack, tint)
                 });
                 elem.parentNode.spliceContent(elem.parentNode.children.indexOf(maskedElems[0]), maskedElems.length, maskGroup);
             }
+
+            // Remove original mask element
+            elem.parentNode.spliceContent(elem.parentNode.children.indexOf(elem), 1, []);
```

Comment:
Here we remove the original mask element after it has been copied over to all elements referencing it.